plugins/controlboard/src/ControlBoard.cpp

```diff
@@ -132,6 +132,25 @@ void ControlBoard::Configure(const Entity& _entity,
         return;
     }
 
+    auto& coupling_group_bottle = m_pluginParameters.findGroup("COUPLING");
+    if (!coupling_group_bottle.isNull()) {
+        m_pluginParameters.unput("device");
+        auto coupling_device_str = coupling_group_bottle.find("device").asString();
+        m_pluginParameters.put("device", coupling_device_str);
+        if(!m_coupling_driver.open(m_pluginParameters))
+        {
+            yError() << "gz-sim-yarp-controlboard-system Plugin failed: error in opening yarp "
+                        "coupling driver";
```

Comment:
```suggestion
                        "coupling driver with device name " << coupling_device_str 
```

I think this may be useful for debug if something does not work.